scorecard/profile_data.py

```diff
@@ -7,22 +8,22 @@
 
 class MuniApiClient(object):
     def __init__(self, geo_code):
-        self.API_URL = 'https://data.municipalmoney.org.za/api/cubes/'
-        if settings.DEBUG:
-            self.API_URL = 'http://127.0.0.1:8888/api/cubes/'
-
         self.geo_code = str(geo_code)
         self.line_item_params = self.get_line_item_params()
 
         self.results = defaultdict(dict)
         self.years = set()
+        responses = []
+        self.session = FuturesSession()
```

Comment:
If we create one MuniApiClient for every request, then this means we'll have a new thread pool for every request. That might be bad, causing a large number of threads. We may want to create a shared thread pool of up to X threads (for some sane X).